src/OrchardCore.Modules/OrchardCore.Shortcodes/Views/Admin/Create.cshtml

```diff
@@ -112,16 +112,16 @@
 
     <div class="form-group">
         <div class="btn-group">
-            <button type="submit" name="submit" class="btn btn-success">@T["Save"]</button>
+            <button type="submit" name="submit" class="btn btn-success save">@T["Save"]</button>
             <button type="button" class="btn btn-success dropdown-toggle dropdown-toggle-split" data-reference="parent" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                 <span class="sr-only">@T["Toggle Dropdown"]</span>
             </button>
             <div class="dropdown-menu">
-                <button class="dropdown-item" name="submit" type="submit" value="SaveAndContinue">@T["and continue"]</button>
+                <button class="dropdown-item save-continue" name="submit" type="submit" value="SaveAndContinue">@T["and continue"]</button>
             </div>
         </div>
 
-        <a class="btn btn-secondary" asp-route-action="Index">@T["Cancel"]</a>
+        <a class="btn btn-secondary cancel" asp-route-action="Index">@T["Cancel"]</a>
```

Comment:
@Skrypt Should all these cancel buttons have role="button" ? I did not think so because they are links to a page. But maybe for a11y?

Reply:
Role is an attribute for accessibility so I think we should have it.

Reply:
Ok I will add them all in this pr then.